### PR TITLE
Remove Mentions of Namespace CLI Configuration Requirements

### PR DESCRIPTION
**Pull request by dubeejw**
- Do not inform the user that he or she must configure a namespace before using the CLI
- Remove Python CLI mentions

Closes https://github.com/openwhisk/openwhisk/issues/1797

### CODE REVIEW

**Comment by dubeejw** [comment]
@csantanapr, please review.

**Comment by rabbah** [comment]
@dubeejw @csantanapr should the CLI be updated to also remove the --namespace features? (property set/get etc.)

**Comment by dubeejw** [comment]
Do you think there are users still out there that haven't unset their namespaces?

**Comment by rabbah** [comment]
If they haven't it's fixable by editing the .wskprops file.

**Comment by dubeejw** [comment]
@csantanapr, is this ready for merge?

**Comment by rabbah** [comment]
rebase this to remove the merges?

**Comment by rabbah** [comment]
Given #1570 - maybe i'll merge this into my pr so i can amend it for --insecure. What do you think?

File: docs/README.md [hunk 1]
```diff
@@ -26,24 +26,23 @@ There are three properties to configure the CLI with:
 
 1. **API host** (name or IP address) for the OpenWhisk deployment you want to use.
 2. **Authorization key** (username and password) which grants you access to the OpenWhisk API.
-3. **Namespace** where your OpenWhisk assets are stored.
 
 The CLI will usually have an API host already set. You can check its value with
```
Comment (rabbah) [inline docs/README.md]: this is not true anymore.

File: docs/README.md [hunk 2]
```diff
@@ -26,24 +26,23 @@ There are three properties to configure the CLI with:
...
-you will need to provide one or both for most CLI operations.
+If you know your authorization key, you can configure the CLI to use it. Otherwise,
+you will need to obtain an authorization key for most CLI operations.
 
 ```
-wsk property set [--apihost <openwhisk_baseurl>] --auth <username:password> --namespace <namespace>
+wsk property set [--apihost <openwhisk_baseurl>] --auth <username:password>
 ```
 
 The API host is set automatically when you build the CLI for your environment. A _guest_ account is available
```
Comment (rabbah) [inline docs/README.md]: and neither is this.

**Comment by rabbah** [comment]
suggest we remove the section: ` ## Setting up the deprecated OpenWhisk CLI (Python based)`.

File: docs/README.md [hunk 3]
```diff
@@ -26,24 +26,23 @@ There are three properties to configure the CLI with:
```
Comment (rabbah) [inline docs/README.md]: "three" is no longer true - about about "some".

File: docs/README.md [hunk 4]
```diff
@@ -26,24 +26,23 @@ There are three properties to configure the CLI with:
...
 2. **Authorization key** (username and password) which grants you access to the OpenWhisk API.
-3. **Namespace** where your OpenWhisk assets are stored.
 
 The CLI will usually have an API host already set. You can check its value with
 `wsk property get --apihost`.
 
-If you know your authorization key and namespace, you can configure the CLI to use them. Otherwise
-you will need to provide one or both for most CLI operations.
+If you know your authorization key, you can configure the CLI to use it. Otherwise,
+you will need to obtain an authorization key for most CLI operations.
```
Comment (rabbah) [inline docs/README.md]: we should link here to how you obtain it - link to wskadmin user create doc? @csantanapr thoughts?

**Comment by rabbah** [comment]
there are also references to setting namespace in the vagrant doc.

File: docs/README.md [hunk 5]
```diff
@@ -22,47 +22,29 @@ A list of downloadable CLIs for various operating systems, and CPU architectures
...
 1. **API host** (name or IP address) for the OpenWhisk deployment you want to use.
 2. **Authorization key** (username and password) which grants you access to the OpenWhisk API.
-3. **Namespace** where your OpenWhisk assets are stored.
 
-The CLI will usually have an API host already set. You can check its value with
-`wsk property get --apihost`.
-
-If you know your authorization key and namespace, you can configure the CLI to use them. Otherwise
-you will need to provide one or both for most CLI operations.
+The API host can be acquired from the `edge.host` property in `whisk.properties` file, which is generated during
```
Comment (dubeejw) [inline docs/README.md]: There maybe an easier way to get the API host?

**Comment by dubeejw** [comment]
Updates added. Will need another review.

File: tools/vagrant/README.md [hunk 6]
```diff
@@ -168,24 +167,6 @@ vagrant ssh
 wskadmin user create <subject>
 ```
 
-This command will create a new *subject* with the authorization key shown on the console once you run `wskadmin`. This key is required when making API calls to OpenWhisk, or when using the command line interface (CLI). The namespace is the same as the `<subject>` name used to create the key.
```
Comment (rabbah) [inline tools/vagrant/README.md]: why is this deleted? this is still necessary.
do you want to move it somewhere else?

Comment (dubeejw) [inline tools/vagrant/README.md]: Looks like the same command is shown in the doc above. Seemed like the one removed was specific to namespaces.

**Comment by dubeejw** [comment]
@csantanapr, please review.

File: docs/README.md (hunk 5 above)
Comment (rabbah) [inline docs/README.md]: link to the ansible file that defines the API host.

File: tools/vagrant/README.md (hunk 6 above)
Comment (rabbah) [inline tools/vagrant/README.md]: can you point to where you're looking for the duplicate text?

Comment (dubeejw) [inline tools/vagrant/README.md]: https://github.com/dubeejw/openwhisk/blob/6277fce44d76730ec6aa745a1727fff3c5349800/tools/vagrant/README.md#adding-openwhisk-users-optional

File: docs/README.md (hunk 5 above)
Comment (dubeejw) [inline docs/README.md]: I think the API host is dependent on the environment in which the deployment is happening.

Comment (rabbah) [inline docs/README.md]: good point - so then no :( although i wouldn't use edge.host, use api.host (in the property file).

File: tools/vagrant/README.md (hunk 6 above)
Comment (rabbah) [inline tools/vagrant/README.md]: that just adds one subject with one namespace.
the text here allows you to add namespaces to the same subject.
it should be restored (although i think all these text should be moved really why is it in the vagrant readme; separate pr).

**Comment by csantanapr** [comment]
@dubeejw did you squashed with new content?

It makes it difficult to review and see what's the delta from the last time you got a review.

Leave the previous commits as you make changes just add more commits, we will take care of squashing at the time of merging.

**Comment by dubeejw** [comment]
@csantanapr, will do.

**Comment by csantanapr** [comment]
merged bb2c198ccc1aaa024c7ddf87dbd4ac686ca78462
